changelog.md

```diff
@@ -1,13 +1,17 @@
+### Next release
```

Comment:
I'm waiting for https://github.com/Huttopia/console-bundle/issues/12 and https://github.com/Huttopia/console-bundle/issues/13 to write `1.2.0` here.